src/antelope/mocks/AntelopeConfig.ts

```diff
@@ -16,12 +19,261 @@ export class AntelopeWallets {
     }
 }
 
-const config = {
-    notifyNeutralMessageHandler: (message: string) => void 0,
-    localizationHandler: (message: string, params?: Record<string, string>) => message,
-    notifyFailureWithActionHandler: (message: string, action: () => void) => void 0,
-    notifyFailureWithAction: (message: string, params?: { label: string; handler: () => void; }) => void 0,
-};
+export class AntelopeConfig {
+    // @TODO rename this method, it's used for token and NFT transfers as well
+    wrapError(description: string, error: unknown): AntelopeError {
```

Comment:
rename to `transactionError` (I updated this in telos-wallet as part of the nft transfer PR)